src/config/constants.ts

```diff
@@ -8,6 +8,7 @@ import {
 import { ITEM_LAYOUT_MODES } from '../enums';
 
 export const APP_NAME = 'Graasp';
+export const PLATFORM_NAME = 'GraaspBuilder';
```

Comment:
I found it neat that you imported the `name` field of `package.json` in `index.tsx`. Would it make sense to do the same here?

Reply:
I should remove this, as it is not used now.